app/test_app/unit/utils/test_attribute_preprocessor.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+from server.utils import attribute_preprocessor as ap
+
+
+@pytest.fixture(scope='module')
+def create_map(create_organisation, init_database):
```

Comment:
Might be worth adding a simple `AttributeMapAPI` test as well, instead of directly creating an `AttributeMap`

Reply:
Ok will do :)